deploy-config/egress_proxy/notify-api-staging.deploy.acl

```diff
@@ -0,0 +1 @@
+Update this file to force a re-deploy of the egress proxy even when notify-api-staging.<allow|deny>.acl haven't changed
```

Comment:
Good idea! Makes me think: should the deploy workflow check for changes in `deploy-proxy/action.yml` along with `*.acl`??

Reply:
Maybe? but also I could see wanting to trigger it just to get a new version, or something else that wouldn't necessarily update the `action.yml` file, so this feels like a better catch-all mechanism

Reply:
Yeah, definitely. I'm thinking in addition to, not instead of, but it's fine either way.